deployments/forgeops/overlay/docker/forgeops-secrets/forgeops-secrets-image/7.0/config.sh

```diff
@@ -141,8 +141,8 @@ keytoolgen -importpass -alias configstorepwd $(useRandomPass ds-passwords/dirman
 genRSA rsajwtsigningkey 2048
 genRSA selfserviceenctest 2048
 genKey selfservicesigntest HmacSHA256 256
-#genEC es256test 256
-keytool -importcert -file /opt/gen/7.0/es256test.cer ${KS_PROPS} -alias es256test -noprompt
+genEC es256test 256
+keytool -importcert -file /opt/gen/7.0/es256iot.cer ${KS_PROPS} -alias es256iot -noprompt
```

Comment:
Just to clarify: we have reverted `es256Test` to a dynamically created key. There is a new secret alias `es256iot` that contains a CA cert which is used for IoT verification but the underlying cert hasn't changed. So the dynamic secrets package should still work?

Reply:
Yes exactly. I think our examples tests use the secrets package so that should be verified.